Sources/PickBetter/BetterPicker.swift

```diff
@@ -31,7 +31,7 @@ public struct BetterPicker<SelectionBox, ItemContent>: View where SelectionBox:
     ///     - style: `Style` (BetterPickerStyle)
     /// - Returns
     ///     - `some View`
-    public func betterPickerStyle<Style: BetterPickerStyle>(_ style: Style) -> some View {
+    @MainActor public func betterPickerStyle<Style: BetterPickerStyle>(_ style: Style) -> some View {
```

Comment:
Should this also have ViewBuilder since it is producing a view?

Reply:
That function isn't using ViewBuilder functionality. I don't think it would help.